www/styles/pages/meeting/[id]/index.module.css

```diff
@@ -1,3 +1,7 @@
 .meetingButton {
   margin-left: auto;
 }
+
+.section {
+  margin: 1.5em 0 1.5em 0;
+}
```

Comment:
There should be a newline here. Try to look at things before you open up a pull request.

Reply:
lets create a linting rule to add new lines if its such a big deal :) I have seen this in your pull requests as well so it seems like a common issue